metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -421,14 +421,8 @@
          nil)))
     (catch Exception e
       (let [error-message (.getMessage e)
-            error-code (cond
-                         (re-find #"unique constraint.*_CID_REV" error-message)
+            error-code (if (re-find #"unique constraint.* violated" error-message)
                          :revision-id-conflict
-
-                         (re-find #"unique constraint.*_CON_REV" error-message)
-                         :revision-id-conflict
-
-                         :else
                          :unknown-error)]
```

Comment:
How does this actual fix the problem?  I just looks like you made the same code better.

Reply:
For granule ingest conflict, the constraint name in the error message is `METADATA_DB.ASF_GRANULES_CRI`, the new search pattern will catch it. The old pattern is too restricted for no reason.